### PR TITLE
Location - CMS field layout

### LINKED ISSUE #55: Location - CMS Fields layout
After working on a project, I ran into some issues with tab order for Location in the CMS. Simplest solution is not to remove the main tab and use Main for the Info tab. This will help ensure new tabs added through extensions will not be the opening tag when adding/editing.

### PR DESCRIPTION
**Pull request by jsirish**
Locator - code formatting cleanup

fixes #55

### CODE REVIEW

File: code/Location.php [hunk 1]
```diff
@@ -40,12 +39,12 @@ class Location extends DataObject implements PermissionProvider
         'State',
         'Postcode',
         'Country',
+        'Website',
+        'Phone',
+        'EmailAddress',
```
Comment (muskie9) [inline code/Location.php]: Again, just things that really have no bearing on how it works... Do we want to shorten this to `Email`?

File: code/Location.php [hunk 2]
```diff
@@ -88,34 +87,23 @@ function fieldLabels($includerelations = true)
 
     public function getCMSFields()
     {
-
         $fields = parent::getCMSFields();
```
Comment (muskie9) [inline code/Location.php]: Per our conversation earlier, do we want to move this away from `parent::getCMSFields()` since it's a `DataObject`?

**Comment by jsirish** [comment]
re: EmailAddress - maybe. Would effect the ModelAdmin and CSVBulkLoader, and would break older installs like 3 Sheeps. Kinda think it'd be best to grandfather it.

re: `parent::getCMSFields()` - I think that's a good idea, I'll work on that. Would make it easier to rename the `Suburb` field to `City` too if I do it after `$this->extend`

File: code/Location.php [hunk 3]
```diff
@@ -130,9 +125,22 @@ public function validate()
         return $result;
     }
 
+    public function EmailAddress()
+    {
+        Deprecation::notice('2.0', 'Use "$Email" instead.');
```
Comment (muskie9) [inline code/Location.php]: Can you update the deprecation to `'3.0`. After some discussions I recently had, this technically wouldn't go away until we release `3.0` and `2.x` will continue to support this. We'll need to do this for any other deprecations going into `master`.

**Comment by jsirish** [comment]
Just pushed an update with the updated Depreciation message
